### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Andersson007**
Update README

### CODE REVIEW

**Review by markuman — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,12 +1,8 @@
 # community-topics
 
-This repository contains topics that will be discussed in the weekly Ansible Community IRC Meetings.
+This repository contains topics that will be discussed and voted by the Asnible Community and the [Ansible Community Steering Committee](https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html) asynchronously.
```
Comment (gundalow) [inline README.md:3]:
```suggestion
This repository contains topics that will be discussed and voted by the Ansible Community and the [Ansible Community Steering Committee](https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html) asynchronously.
```

File: README.md [hunk 2]
```diff
@@ -16,3 +12,13 @@ Community Meeting summaries/minutes and logs are posted to https://github.com/an
 4. At the meeting, we look at the list of open issues, and issues with `next_meeting` get first priority.
     * Currently we assume that the list of open issues won't be too long, and that not too many will have a `next_meeting` label, so we don't need a fully defined process on how to select topics for the meeting.
     * If we find out that this assumption isn't good, we can re-evaluate/discuss this in a meeting.
+
+## Community meetings
+
+Some of the topics can be additionally discussed in the weekly Community Meetings though final decisions are always made by **asynchronous** voting in the topics.
+
+See the Community Working Group meeting [schedule](https://github.com/ansible/community/blob/main/meetings/README.md#wednesdays) and join us in any of the following bridged channels:
+* on [Matrix](https://docs.ansible.com/ansible/latest/community/communication.html#ansible-community-on-matrix): `#community:ansible.im`
```
Comment (gundalow) [inline README.md:21]: `ansible.com` for links (not `.im)

Also, we should give a Matrix.to link, ie `https://matrix.to/#/#community:ansible.com`

Comment (Andersson007) [reply README.md]: fixed, thanks, ptal

**Review by gundalow — APPROVED** [review]

**Comment by Andersson007** [comment]
@markuman @gundalow thanks for reviewing!

**Pull request merged**